sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,14 +116,14 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 7384e2d7b908de0b2bca9f3c57827ea84698864e
-  # Merge: ed85c3e2 a32549b4
-  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
-  # Date:   Fri Sep 20 14:56:40 2024 +0100
-  #   Merge pull request #2108 from pbalcer/fix-perf-tracing-logs
-  #   fix perf regression in tracing layer
-  set(UNIFIED_RUNTIME_TAG 7384e2d7b908de0b2bca9f3c57827ea84698864e)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/AllanZyne/unified-runtime.git")
+  # commit 7e38af77174f92bf4025122b1fa8e3fc49f3ba29
+  # Merge: ddafd29b 6c7d7a76
+  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
+  # Date:   Mon Jul 15 15:14:47 2024 +0100
+  #     Merge pull request #1863 from kbenzie/benie/main-reverts
+  #     Revert "Merge pull request #1849 from ayylol/cl-subgroupsizes"
+  set(UNIFIED_RUNTIME_TAG review/yang/statistics)
```

Comment:
https://github.com/oneapi-src/unified-runtime/pull/1869 is merged, please update